projects/core/src/model/org-unit.model.ts

```diff
@@ -9,7 +9,7 @@ export interface CostCenter {
   unit?: B2BUnit;
 }
 
-export enum B2BUserGroup {
+export enum B2BUserRole {
   B2B_ADMIN_GROUP = 'b2badmingroup',
```

Comment:
Should the values also be rename to drop the `GROUP` word. It looks strange that we have groups as a role?

Reply:
removed B2B preffix too (redundant)